misc/config_tools/config_app/views.py

```diff
@@ -973,7 +973,7 @@ def get_generic_scenario_config(scenario_config, add_vm_type=None):
             'SOS_VM': ('industry', 'vm:id=0'),
             'POST_STD_VM': ('industry', 'vm:id=1'),
             'POST_RT_VM': ('industry', 'vm:id=2'),
-            'KATA_VM': ('industry', 'vm:id=1'),
+            'KATA_VM': ('industry', 'vm:id=7'),
```

Comment:
Is there any impact here for scenarios that do not list as many VMs?

Reply:
I tried adding kata vm as id `1`, `3` and `7`, all these patterns can be created successfully.
By the way, I also tested in `hybrid`, `hybrid_rt` and `logical partition` scenarios, all passed with just adding kata vm type operation.

Reply:
Did you try to run these or did you just try to generate the scenario XML file?

Reply:
For running kata, I just verified that with `industry` scenario and `vm_id 7`.
For other patterns, just tried to generate those scenario xmls.

Reply:
Can we run an additional test with a scenario that does not have as many VMs (e.g. `hybrid`)?

Reply:
> Is there any impact here for scenarios that do not list as many VMs?

Hi， @gvancuts , here the reason we changed the default kata VM id to 7 is that when we create a new Kata VM in config tool, the config tool will get the template Kata VM config from misc/config_tools/data/generic_board/industry.xml where the VM ID of the Kata VM is 7, and then configure the template Kata VM config to the destination scenario XML. So this change has no impact with the VM id/VM numbers of the scenario we create.

Reply:
> Can we run an additional test with a scenario that does not have as many VMs (e.g. `hybrid`)?

Running kata vm successfully with the `hybrid` scenario. 😄 
```
root@intel-WL10:~# ps -ef | grep acrn| more
root        3802    3764  0 19:35 ?        00:00:02 /usr/bin/acrn-dm -U a7ada506
-1ab0-4b6b-a0da-e513ca9b8c2f -A -m 2048M -s 0:0,hostbridge -s 1:0,lpc -lcom1,std
io -s 3,virtio-console,@socket:console0=/run/vc/vm/80eafdaaac2db5a679c32268840f3
5830e1043fe3fde8b365cd1847b30dc9648/console.sock -s 4,virtio-blk,/opt/kata/share
/kata-containers/kata-containers-image_clearlinux_1.11.1_agent_34a000d0cf.img -s
 5,virtio-blk,nodisk -s 6,virtio-blk,nodisk -s 7,virtio-blk,nodisk -s 8,virtio-b
lk,nodisk -s 9,virtio-blk,nodisk -s 10,virtio-blk,nodisk -s 11,virtio-blk,nodisk
 -s 12,virtio-blk,nodisk -s 13,virtio-console,socket:agent.channel.0=/run/vc/vm/
80eafdaaac2db5a679c32268840f35830e1043fe3fde8b365cd1847b30dc9648/kata.sock -s 14
,virtio-net,tap0_kata,mac=02:42:ac:12:00:02 -k /opt/kata/share/kata-containers/v
mlinuz-5.4.32-73 -B tsc=reliable no_timer_check nohpet console=tty0 console=ttyS
0 console=hvc0 log_buf_len=16M consoleblank=0 iommu=off i915.avail_planes_per_pi
pe=0x070F00 i915.enable_hangcheck=0 i915.nuclear_pageflip=1 i915.enable_guc_load
ing=0 i915.enable_guc_submission=0 i915.enable_guc=0 root=/dev/vda1 rw rootwait
debug systemd.show_status=true systemd.log_level=debug systemd.log_target=kmsg p
rintk.devkmsg=on panic=1 maxcpus=1 systemd.unit=kata-containers.target systemd.m
ask=systemd-networkd.service systemd.mask=systemd-networkd.socket agent.log=debu
g agent.log=debug initcall_debug sandbox-80eafdaaac2db5a679c32268840f35830e1043f
e3fde8b365cd1847b30dc9648
```